packages/slice-machine/src/legacy/lib/builders/SliceBuilder/Sidebar/index.tsx

```diff
@@ -73,32 +73,40 @@ export const Sidebar: FC = () => {
             variationId={v.id}
           />
         ))}
-        <Box
-          backgroundColor="grey2"
-          bottom={0}
-          flexDirection="column"
-          padding={{ bottom: 40, inline: 24 }}
-          position="sticky"
-          // As `PageLayoutContent` has a `16px` bottom padding, we need to
-          // apply an equal negative margin to the `Box` if we want it to sit
-          // flush with the bottom of the page.
-          // @ts-expect-error TODO(PBD-1080): write the new `ScrollAreaEndGradient` component instead of using a `Box`.
-          style={{ marginBottom: "-16px" }}
+        {/*
+         * As `PageLayoutContent` has a `16px` bottom padding, we need to apply
+         * an equal negative margin to the `div` if we want it to sit flush with
+         * the bottom of the page.
+         * TODO(PBD-1080): write the new `ScrollAreaEndGradient` component
+         * instead of using a `div`.
+         */}
+        <div
+          style={{
+            bottom: theme.space[0],
+            marginBottom: `-${theme.space[16]}`,
+            position: "sticky",
+          }}
```

Comment:
❓ How do we decide whether to use a CSS class or a `style` prop?

❓ How do we decide when to use `theme` (e.g. `theme.space[16]`) or hard-coded values (e.g. 16px)?